src/main/hover/goHover.ts

```diff
@@ -28,4 +32,79 @@ export class GoHover extends AbstractHoverProvider {
         }
         return undefined;
     }
+
+    provideHover(document: vscode.TextDocument, position: vscode.Position): vscode.ProviderResult<vscode.Hover> {
+        let node: DependenciesTreeNode | undefined = this.getNodeByLocation(document, position);
+        if (!node || !(node instanceof GoDependenciesTreeNode)) {
+            return;
+        }
+        const credentialsSet: boolean = this._treesManager.connectionManager.areCredentialsSet();
+        let hoverText: vscode.MarkdownString = new vscode.MarkdownString(
+            `${node.componentMetadata?.description ? `${node.componentMetadata.description.slice(0, 100)}` : ''}${
+                node.componentMetadata.description?.length > 100 ? ` ...` : ` `
+            }[ReadMe](${node.componentMetadata?.gocenter_readme_url})${
+                node.componentMetadata?.gocenter_metrics_url ? ` | [Metrics-Info](${node.componentMetadata.gocenter_metrics_url})\n\n` : '\n\n'
+            }`
+        );
+        if (credentialsSet) {
+            hoverText.appendMarkdown(
+                this.createLatestVersionText(node.componentMetadata?.latest_version) + this.createLicensesText(this.licensesToMarkdown(node.licenses))
+            );
+        } else {
+            if (node.topIssue.severity === Severity.Normal) {
+                hoverText.appendMarkdown(
+                    `[No vulnerabilities found](${GoHover.XRAY_IN_GOCENTER_URL})\n\n${this.createLatestVersionText(
+                        node.componentMetadata?.latest_version
+                    )}` + this.createLicensesText(node.componentMetadata?.licenses)
+                );
+            } else {
+                let [summary, issuesCount] = this.getSeveritySummary(node.componentMetadata?.vulnerabilities?.severity);
+                hoverText.appendMarkdown(
+                    `${this.createLatestVersionText(node.componentMetadata?.latest_version)}` +
+                        this.createLicensesText(node.componentMetadata?.licenses) +
+                        `${issuesCount ? `${issuesCount}\n\n` : ``}` +
+                        `${
+                            summary
+                                ? `${summary} | CVE information available in: [JFrog GoCenter](${node.componentMetadata?.vulnerabilities?.gocenter_security_url})\n\n`
+                                : ``
+                        } `
+                );
+            }
+        }
+        return new vscode.Hover(hoverText);
+    }
+
+    /**
+     * Creates two strings from 'ISeverityCount', both are in markdown syntax.
+     * The first string is one line contains all the severities with an icon corresponding to the severities level,
+     * the second is the sum of all severities..
+     */
```

Comment:
If you add JSDoc, at least make it complete.
```suggestion
     * @param severities - bla bla
     * @returns bla bla
     */
```